tests/bound.rs

```diff
@@ -0,0 +1,25 @@
+// Copyright 2017 The CGMath Developers. For a full listing of the authors,
+// refer to the Cargo.toml file at the top-level directory of this distribution.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+extern crate cgmath;
+extern crate collision;
+
+use collision::Bound;
+
+fn _box(_: Box<Bound<f32>>) {}
+
+#[test]
+fn bound_box() {
```

Comment:
Can you just leave a comment that this is testing if the `Bound` trait can be boxed or not. It's rather counter intuitive as it is right now.